app/static/celebrity.js

```diff
@@ -0,0 +1,117 @@
+function getCelebs(dateToday) {
+    const celebCards = document.getElementById("celebCards");
+
+    fetch('https://www.imdb.com/search/name/?birth_monthday=' + dateToday)
+        .then(function (response) {
+            return response.text();
+        }).then(function (html) {
+            celebCards.innerHTML = "";
+            const parser = new DOMParser();
+            const doc = parser.parseFromString(html, 'text/html');
+
+            function getNames() {
+                let namesArray = new Array();
+                const names = doc.querySelectorAll('h3.lister-item-header');
+                for (let i = 0; i < 12; i++) {
+                    namesArray.push(names[i].innerText.trim().substr(5).trim());
+                }
+                return namesArray;
+            }
+
+            function getImages() {
```

Comment:
Looks like you can create a more abstract version of `getNames`, `getImages` and `getJobs`